packages/design-system/src/styles/components/_Table.scss

```diff
@@ -30,26 +30,32 @@ $table-stacked-header-max-width: 160px !default;
   th {
     border: 0;
     border-top: 1px solid;
+
+    /* autoprefixer grid: on */
     display: grid;
     // CSS grid doesn't support word-wrap. Use minmax as a workaround
     // https://css-tricks.com/css-grid-in-ie-debunking-common-ie-grid-misconceptions/#fit-content-is-not-ie-friendly-but
     grid-template-columns: minmax(0, $table-stacked-header-max-width) 1fr;
     position: relative;
 
-    // stylelint-disable selector-no-qualifying-type
+    /* stylelint-disable-next-line selector-no-qualifying-type */
     &[data-title]::before {
```

Comment:
It would help to leave a comment explaining what this psuedo element is and what its for

Reply:
[Here's](https://github.com/CMSgov/design-system/pull/853/files#diff-2b709213c3173751c76a6658e5db4facf82984bc78f95a2598faf72564b4ebcfL63) another example of an area that could benefit from a comment